server/src/main/java/edu/jhu/server/Game.java

```diff
@@ -60,9 +87,26 @@ public void handleEvent(JSONObject event) {
 	  		System.out.println("test event");
 	  		break;
 	  	case CHAT_NOTIFICATION:
-	  		System.out.println(event.getString("body"));
 	  		notifyPlayers(event);
 	  		break;
+	  	case SUGGESTION_REQUEST:
```

Comment:
Hmm, maybe have a separate method for each event type here? just so this method doesn't get huge. Also then probably wouldn't need the makeChatMessage method.